README.md

```diff
@@ -8,6 +8,10 @@
 
 An asynchronous effect monad and threading model for PureScript.
 
+`aff` is in the same space as JavaScript Promises, but with additional support for cancellation (async exceptions) and resource management (`bracket`). It’s cooperative multi-tasking which yields on async constructors (`makeAff`).
+
+For OS-thread parallelism in Node.js, see [node-workerbees](https://github.com/natefaubion/purescript-node-workerbees).
```

Comment:
Since node-workerbees is experimental, I’m not sure we should link it here. What do you think?

Reply:
I don't have a strong opinion about that.

Reply:
Maybe a __Related Packages__ section further down in the README?

Reply:
We should probably put this in related packages too: https://pursuit.purescript.org/packages/purescript-avar/4.0.0